src/nft/hooks/useUsdPrice.ts

```diff
@@ -1,14 +1,14 @@
 import { formatEther } from '@ethersproject/units'
 import { ChainId } from '@uniswap/sdk-core'
-import { useLocalCurrencyPrice } from 'hooks/useLocalCurrencyPrice'
+import { useUSDPrice } from 'hooks/useUSDPrice'
 import useNativeCurrency from 'lib/hooks/useNativeCurrency'
 import tryParseCurrencyAmount from 'lib/utils/tryParseCurrencyAmount'
 import { GenieAsset } from 'nft/types'
 
 export const useNativeUsdPrice = (chainId: number = ChainId.MAINNET): number => {
   const nativeCurrency = useNativeCurrency(chainId)
   const parsedAmount = tryParseCurrencyAmount('1', nativeCurrency)
-  const usdcValue = useLocalCurrencyPrice(parsedAmount)?.data ?? 0
+  const usdcValue = useUSDPrice(parsedAmount)?.data ?? 0
   return usdcValue
 }
```

Comment:
would it be possible to remove this file and use 'hooks/useUSDPrice' directly instead?